app/models/organisation/organisation_type_concern.rb

```diff
@@ -74,6 +74,11 @@ def active_child_organisations_excluding_sub_organisations_grouped_by_type
       active_child_organisations_excluding_sub_organisations.group_by(&:organisation_type).sort_by { |type, department| type.listing_position }
   end
 
+  def active_child_organisations_excluding_sub_organisations_and_courts_and_tribunals
+    @active_child_organisations_excluding_sub_organisations ||=
+      child_organisations.excluding_govuk_status_closed.excluding_courts_and_tribunals.with_translations(I18n.locale).where("organisation_type_key != 'sub_organisation'").ordered_by_name_ignoring_prefix
```

Comment:
Is it worth pulling the `.where("organisation_type_key != 'sub_organisation'")` into a scope on `Organisation`?